packages/katello/rubygem-foreman_scc_manager/rubygem-foreman_scc_manager.spec

```diff
@@ -113,6 +112,13 @@ cp -a .%{gem_dir}/* \
 %{gem_instdir}/test
 
 %changelog
+* Wed Jan 11 2023 Nadja Heitmann <nadjah@atix.de> 2.2.0-1
+- Update to 2.2.0
+- Fix permission issue for new installations
+
+* Mon Nov 14 2022 Bernhard Suttner <suttner@atix.de> 2.1.0-1
+- Update to 2.1.0
```

Comment:
This was a different commit. What we always do is cherry pick both commits so you get the correct log/references. You can feed `git cherry-pick -x` multiple commits to make this easier.

Reply:
> This was a different commit. What we always do is cherry pick both commits so you get the correct log/references. You can feed `git cherry-pick -x` multiple commits to make this easier.

Thanks for the hint, changed the PR accordingly! Is it still possible to CP something to 'rpm/3.3 branch'?

Reply:
Yes. We still have the pipelines in place even though we officially ended support for 3.3.